docs/Tutorials/Quickstarts/Private-Network-Quickstart.md

```diff
@@ -430,7 +430,7 @@ Once complete and successful, the status of the pet you adopted will show 'Succe
 
 ![Dapp UI](../../images/dapp-ui.png)
  
-You can also search for the transaction and view its details in the Block Explorer UI 
+You can also search for the transaction and view its details in the Block Explorer UI at `http://localhost:25000/`
```

Comment:
if you don't use code back ticks, the link will be clickable.

Reply:
:+1: